src/frontend/components/contribute/step-2/sub-steps/already-have-wallet.tsx

```diff
@@ -49,10 +51,16 @@ export class AlreadyHaveWallet extends React.Component<IAlreadyHaveWalletProps,
         }
     }
 
-    private handleWalletAddressChange = () => {};
-    private handleContinue = () => {
-        setSubStep(-1);
-        incrementStep();
+    private handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setState({ targetAddress: event.target.value });
```

Comment:
I've already done this... In fact I used a function that I would import because it's used more than once.